src/main/java/com/eleks/mentorship/MentorshipApplication.java

```diff
@@ -1,13 +1,28 @@
 package com.eleks.mentorship;
 
+import jakarta.annotation.PreDestroy;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import com.opentable.db.postgres.embedded.EmbeddedPostgres;
+
+import java.io.IOException;
 
 @SpringBootApplication
 public class MentorshipApplication {
 
-	public static void main(String[] args) {
+	private static EmbeddedPostgres postgres;
+
+	public static void main(String[] args) throws IOException {
+		postgres = EmbeddedPostgres
+				.builder()
+				.setPort(5433)
```

Comment:
could be moved to properties file

Reply:
If you mean "spring.datasource.url" property, then it is already there. We can't remove this code, as we need it to create an embedded database, that will exist until the app is running. Or do you mean another approach here that I am not acknowledged of?

Reply:
no, I am talking about the port number :)